dg_export.py

```diff
@@ -23,6 +22,6 @@ def parser_arguments():
 
     model = YOLO(args.weights)
 
-    success = model.export(format=args.format, imgsz=args.imgsz, data=args.data, int8=args.quantize, separate_outputs=True, export_hw_optimized=True, uint8_io_dtype=True, max_ncalib_imgs=args.max_ncalib_imgs)
+    success = model.export(format=args.format, simplify=True, imgsz=args.imgsz, data=args.data, int8=args.quantize, separate_outputs=True, export_hw_optimized=True, uint8_io_dtype=True, max_ncalib_imgs=100)
```

Comment:
can you add back this line?

    parser.add_argument('--max_ncalib_imgs', type=int, default=300, help='calibration image set size')	
  
make default = 100